src/scales/scale.logarithmic.js

```diff
@@ -69,19 +62,20 @@ const defaultConfig = {
 class LogarithmicScale extends Scale {
 	_parse(raw, index) { // eslint-disable-line no-unused-vars
 		const value = LinearScaleBase.prototype._parse.apply(this, arguments);
-		return helpers.isFinite(value) && value >= 0 ? value : undefined;
+		if (value === 0) {
```

Comment:
I think I prefer calculating `minPositive` in `_getMinMax` rather than handling `0` specially here. (Actually probably what would be nicest is overriding `_getMinMax` in the log scale, but the `_getMinMax` logic has gotten so complicated that's a bit harder). 

E.g. what if someone has a toggle between linear and log scales and wants to set `parsing: false`? I think the more we can keep scales out of the parsing business the better.

Reply:
Data with only positive values (and 0) works fine without parsing, if its in the correct format. One needs to set min value to scale manually then, but that would be expected when parsing=false, wouldn't it?
I have a follow-up to make `beginAtZero` work nicely on log scale, but I'm getting bored with this.

Reply:
In my mind `parsing` is just that. It doesn't include validation or anything else. But I don't necessarily want to spend more time on this, so if you and @etimberg are both okay with this then so am I